TYPO3.Neos/Documentation/CreatingASite/NodeTypes/NodeTypeDefinition.rst

```diff
@@ -69,6 +69,19 @@ The following options are allowed:
             # only allow images in this ContentCollection
             'TYPO3.Neos.NodeTypes:Image': TRUE
             '*': FALSE
+  By using ``position``, it is possible to define the order in which child nodes appear in the structure tree.
```

Comment:
even though rendering works, an empty line should be put here to make the separate blocks easier distinguishable